README_sqlite_dbs.md

```diff
@@ -7,12 +7,27 @@ This directory contains SQLite databases that make our clean tabular
 There are DBs for the versions of the MCRF OMOP CDM data, the CDM
 vocabulary, and examples for various classification tasks.
 
+### naming convention
```

Comment:
I don't think the naming convention should be front and center. Indeed, I believe it should be last as it only concerns people who will build DBs.